app/schemas.py

```diff
@@ -56,20 +57,64 @@ class LocationBase(LocationCreate):
     osm_address_country: str | None
     osm_lat: float | None
     osm_lon: float | None
-    created: datetime
-    updated: datetime | None
+    created: datetime.datetime
+    updated: datetime.datetime | None
 
 
 class PriceCreate(BaseModel):
     model_config = ConfigDict(from_attributes=True, arbitrary_types_allowed=True)
-
-    product_code: str = Field(min_length=1, pattern="^[0-9]+$")
-    price: float
-    currency: str | Currency
-    location_osm_id: int = Field(gt=0)
-    location_osm_type: LocationOSMType
-    date: date
-    proof_id: int | None = None
+    product_code: str | None = Field(
+        default=None,
+        min_length=1,
+        pattern="^[0-9]+$",
+        description="barcode (EAN) of the product, as a string.",
+        examples=["16584958", "1234567890123"],
+    )
+    category_tag: str | None = Field(
```

Comment:
in the long terme could this be an enum from the openfoodfacts-python library ?

Reply:
Not really, as the category taxonomy is frequently updated.